extensions/amp-connatix-player/validator-amp-connatix-player.protoascii

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-connatix-player
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-connatix-player"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-connatix-player>
+  html_format: AMP
+  tag_name: "AMP-CONNATIX-PLAYER"
+  requires_extension: "amp-connatix-player"
+  attr_lists: "extended-amp-global"
+  attrs: {
+    name: "data-player-id"
+    mandatory: true
+  }
+  attrs: {
```

Comment:
All attributes starting with `data-` are automatically valid. It's not necessary to list them unless there are additional restrictions on the attribute. Please remove `data-media-id` since there are no other restrictions on it.

Reply:
fixed